EventEnabledInsurance/readme.md

```diff
@@ -496,3 +496,76 @@ To delete the topic.
 - Click `Topics` on the left hand side
 - Click the `...` menu for the `sor.public.quotes` topic
 - Choose `Delete this topic`, then `Delete`
+
+# REST endpoint
+
+## ACE flow
+
+### GET
+![get sub flow](./media/rest-get-flow.png)
+
+### POST
+![post sub flow](./media/rest-post-flow.png)
+
+The pipeline deploys an ACE integration server (`ace-rest-int-srv-eei`) that hosts the `/eventinsurance/quote` endpoint. The route of the integration server can be found with `oc get -n $NAMESPACE route ace-rest-int-srv-eei-https -ojsonpath='.spec.host'`. Make sure to use HTTPS.
```

Comment:
I think this should also mention APIC, and the testing should be done using the APIC entry point. I.e.:
```
API_BASE_URL=$(oc get secret eei-api-endpoint-client-id -o jsonpath='{.data.api}' | base64 --decode)
API_CLIENT_ID=$(oc get secret eei-api-endpoint-client-id -o jsonpath='{.data.cid}' | base64 --decode)

curl "${API_BASE_URL}/quote" -H "X-IBM-Client-Id: ${API_CLIENT_ID}" -k \
  -d $'{\n  "name": "Ronald McGee",\n  "email": "zarhuci@surguf.zm",\n  "age": 68221627,\n  "address": "408 Uneit Manor",\n  "usState": "CT",\n  "licensePlate": "hezihe",\n  "descriptionOfDamage": "56"\n}'
```